server/api/routes/sqlalchemy/question.py

```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+sys.path.append(os.path.realpath('.'))
+from flask import Blueprint, request
+from models.models import Question, QuestionSchema, db
+
+question_schema = QuestionSchema()
+question_routes = Blueprint("question_routes", __name__)
+
+# Below we have "candidateid" hardcoded, this should change in the future
+@question_routes.route('questionId/question', methods=['POST'])
+def add_question():
+    questionText = request.json['questionText']
+    questionId = request.json['questionId']
+ 
+    new_question = Question(questionText, questionId)
+ 
+    db.session.add(new_question)
+    db.session.commit()
+
+    return question_schema.jsonify(new_question)
+
+@question_routes.route('/<formId>/questions', methods=['GET'])
+def get_question(formId):
+    product = Question.query.get(id)
```

Comment:
id is not defined anywhere, I think you mean formId which gets passed in in the function definition.